src/main/java/hello/controller/IssueController.java

```diff
@@ -0,0 +1,183 @@
+        // send post request to codegen
+        try {
+            final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
+
+            OkHttpClient client = new OkHttpClient();
+
+            okhttp3.RequestBody body = okhttp3.RequestBody.create(JSON, json);
+            Request request = new Request.Builder().url(url).post(body).build();
+            Response response = client.newCall(request).execute();
+
+            return new Gson().fromJson(response.body().string(), CodeGenPostResponse.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new CodeGenPostRequestException("Some Error Occured in Codegen Post Request");
+        }
+    }
+
+    private CodeGenGetResponse sendGetRequestCodeGen(String url) {
+        // send get request to codegen
+        try {
+            OkHttpClient client = new OkHttpClient();
+
+            Request request = new Request.Builder().url(url).build();
+            Response response = client.newCall(request).execute();
+
+            return new Gson().fromJson(response.body().string(), CodeGenGetResponse.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return new CodeGenGetResponse("Failure", "");
+        }
+    }
+
+    private void fixIssue1(Integer methodId, String modifiers) {
+        String [] modifierList = {"public", "protected", "private", "static", "abstract", "synchronized", "transient", "volatile", "final", "native", "strictfp"};
+        String newModifiers = "";
+        for(String modifier : modifierList) {
+            if (modifiers.contains(modifier)) {
+                newModifiers += modifier + ", ";
+            }
+        }
+        if (newModifiers.endsWith(", ")) {
+            newModifiers = newModifiers.substring(0, newModifiers.length() - 2);
+        }
+        newModifiers = "[" + newModifiers + "]";
+        Session session = driver.session();
+        session.run("match (n) where id(n)=" + methodId + " set n.modifiers=\"" + newModifiers + "\"");
+    }
+
+    private void fixIssue3(Integer methodId, String modifiers) {
+        String [] modifierList = {"public", "protected", "private", "static", "abstract", "synchronized", "transient", "volatile", "final", "native", "strictfp"};
+        String newModifiers = "";
+        for(String modifier : modifierList) {
+            if (modifiers.contains(modifier) || modifier.equals("synchronized")) {
+                newModifiers += modifier + ", ";
+            }
+        }
+        if (newModifiers.endsWith(", ")) {
+            newModifiers = newModifiers.substring(0, newModifiers.length() - 2);
+        }
+        newModifiers = "[" + newModifiers + "]";
+        Session session = driver.session();
+        session.run("match (n) where id(n)=" + methodId + " set n.modifiers=\"" + newModifiers + "\"");
+    }
+
+    @GetMapping(path="/fix", produces = "application/json")
```

Comment:
Logic looks good but the method seems to be too long.
We can move the logic inside each switch case into a separate method to shorten the method a bit.